db/migrate/20160301113717_add_confirmable_attributes_to_users.rb

```diff
@@ -11,7 +11,7 @@ def change
     add_index :users, :confirmation_token,   unique: true
 
     if ENV['REQUIRE_CONFIRMED_EMAIL'] != 'true' && ActiveRecord::Base.connection.column_exists?(:users, :confirmed_at)
-      User.update_all('confirmed_at = NOW()')
+      User.update_all(:confirmed_at => DateTime.now)
```

Comment:
Can you change this to `confirmed_at: Time.zone.now`? That ensure ActiveRecord handles the possibly needed time zone conversion.